index.bs

```diff
@@ -123,22 +125,16 @@ Issue: We should consider having further normative restrictions on file names th
 never be allowed using this API, rather than leaving it entirely up to underlying file
 systems.
 
-A <dfn export id=file>file entry</dfn> additionally consists of
-<dfn for="file entry" export>binary data</dfn> (a [=byte sequence=]), a
-<dfn for="file entry">modification timestamp</dfn> (a number representing the number of milliseconds since the <a spec=FileAPI>Unix Epoch</a>),
-a <dfn for="file entry">lock</dfn> (a string that may exclusively be "`open`", "`taken-exclusive`" or "`taken-shared`")
-and a <dfn for="file entry">shared lock count</dfn> (a number representing the number shared locks that are taken at a given point in time).
-
-A user agent has an associated <dfn>file system queue</dfn> which is the
+A user agent has an associated <dfn>file system queue</dfn>  which is the
 result of [=starting a new parallel queue=]. This queue is to be used for all
 file sytem operations.
 
 <div algorithm>
-To <dfn for="file entry/lock">take</dfn> a [=file entry/lock=] with a |value| of
-"`exclusive`" or "`shared`" on a given [=file entry=] |file|:
+To <dfn for="file system entry/lock">take</dfn> a [=file system entry/lock=] with a |value| of
+"`exclusive`" or "`shared`" on a given [=/file system entry=] |entry|:
 
-1. Let |lock| be the |file|'s [=file entry/lock=].
-1. Let |count| be the |file|'s [=file entry/shared lock count=].
+1. Let |lock| be the |entry|'s [=file system entry/lock=].
+1. Let |count| be the |entry|'s [=file system entry/shared lock count=].
```

Comment:
This doesn't take into account locked children. Will that be done in a separate PR?

Reply:
I can do it in a follow up PR. Does it make sense to do it that way?

Reply:
For now this PR is just a mechanical change to move locking from _file entry_ -> _file system entry_, but as written, it doesn't fully fix #137. Personally, I'd prefer to account for the locked children here, but if you'd prefer we can land this in two steps

If so, we should update the PR description to better describe that this is just a mechanical change, and add something like `Issue(137): Taking a lock on a directory should only succeed if no contained file is locked.` in the algorithm